### ISSUE TITLE
Use solid background in call to reduce GPU load

### ISSUE THREAD

**Issue by SystemKeeper** [nextcloud/spreed#11005]
@nextcloud/designers I think we should just go back to a solid background here. It really doesn't make sense to have a setting for this.

_Originally posted by @marcoambrosini in https://github.com/nextcloud/spreed/issues/10464#issuecomment-1714818063_

**Comment by SystemKeeper** [comment]
Or are you talking about blur vs. transparency vs. solid?

**Comment by Antreesy** [comment]
> Or are you talking about blur vs. transparency vs. solid?

Exactly, just like it works now (disabled state) vs. complete solid background.
Theoretically, browser won't have to compute the pixels color (image + semi-transparent overlay), and use only solid overlay color, so it's up to be tested.

**Comment by marcoambrosini** [comment]
I think that not blurring and keeping the transparency can reult in too sharper edges that will make the user interface elss readable depending on the background, so I think we shouldn't do that. 

Another solution could be swapping the regular background for a server-generated blurred version of it that is displayed everywhere but in the dashboard app.
This will also help make the header more readable when an image that has sharp contrasts in that area is used.
e.g:
![Screenshot from 2023-12-04 09-22-32](https://github.com/nextcloud/spreed/assets/26852655/a19c7bc8-c62e-4ba0-a394-72b26e12cfd1)

cc @nextcloud/designers

**Comment by Antreesy** [comment]
> it's up to be tested.

Can't track any difference in GPU load between solid and semi-transparent background, both show the same result (around 50-60% of load when blur is enabled). So it's fine to consider that both options are good to go
